Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -569,10 +569,17 @@ class BrowserViewController: UIViewController {
                                                          object: nil)
     }
 
-    fileprivate func hasPendingCrashReport() -> Bool {
+    fileprivate func crashedLastLaunch() -> Bool {
         return SentryIntegration.crashedLastLaunch
     }
 
+    fileprivate func cleanlyBackgrounded() -> Bool {
+        guard let appDelegate = UIApplication.shared.delegate as? AppDelegate else {
+            return false
```

Comment:
Yeah, "should never happen" - but if it does (???) we default to not showing a dialog.